packages/extension-browser/webpack.config.js

```diff
@@ -83,6 +83,7 @@ module.exports = (env) => {
         resolve: {
             alias: {
                 '@hint/utils/dist/src/network/request-async$': path.resolve(__dirname, 'dist/src/shims/request-async.js'),
+                'axe-core': path.resolve(__dirname, '../../node_modules/axe-core/axe.min.js'), // yarn 1.16 hoists the beta version
```

Comment:
If the package `yarn` hoists changes on us we probably won't notice. Maybe using `require` we can ensure we pull whichever version is associated with this package?
```suggestion
                'axe-core': require.resolve('axe-core/axe.min.js'),
```